endpoints/openrtb2/amp_auction.go

```diff
@@ -154,7 +154,8 @@ func (deps *endpointDeps) AmpAuction(w http.ResponseWriter, r *http.Request, _ h
 	} else {
 		labels.CookieFlag = pbsmetrics.CookieFlagYes
 	}
-	labels.PubID = effectivePubID(req.Site.Publisher)
+	labels.PubID = req.Site.Publisher.ID
```

Comment:
Is there a reason you are simplifying this this way? Currently an amp request should have a site and not an app object, but I don't think it is enforced. Someone at some point might come up with a use case for an app based amp request. This will mean we will break if an amp request uses app (we don't enforce sites in amp as far as I saw). We also won't get the proper parent_id if that was set in the amp request.